sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -52,7 +52,7 @@ trait CheckAnalysis extends PredicateHelper {
   }
 
   protected def mapColumnInSetOperation(plan: LogicalPlan): Option[Attribute] = plan match {
-    case _: Intersect | _: Except | _: Distinct =>
+    case _: Intersect | _: ExceptBase | _: Distinct =>
```

Comment:
then, we do not need these changes.

Reply:
@gatorsmile Thats right Sean. We will not need changes here. However may i request you to please command-B on Except class ? We may need to change the pattern matching in other places, right ? Just wanted to make sure you are okay with it before i went ahead and made the changes.

Reply:
I am fine about that. Please make a change and avoid introducing a new LogicalPlan node.

Reply:
@gatorsmile @maropu OK

Reply:
@gatorsmile @maropu I have removed ExceptAll operator.